nestkernel/event.h

```diff
@@ -736,6 +736,10 @@ class SecondaryEvent : public Event
 
   virtual synindex get_syn_id() const = 0;
 
+  virtual void add_syn_id( const synindex synid ) = 0;
+
+  virtual bool has_syn_id( const synindex synid ) const = 0;
```

Comment:
Maybe `supports_syn_id()` would be an even better name. "Has" suggests that the synapse ID would be unique, but the main point here is that the `GapJEvent` now supports multiple synapse IDs.

I also wonder if we now still need the `get_syn_id()` method. Since the `GapJEvent` now supports multiple IDs, that method cannot return a uniquely defined value (unless one somewhat arbitrarily decides that the first gap junction synapse model defined is special).